README.md

```diff
@@ -1,35 +1,3 @@
 # Spack Config
 
 Shared spack configuration for building software on gadi@NCI
```

Comment:
We should probably have a bit more information here. Who is this repo for? If not for normal users then that's cool, we just need to let them know that.